src/base/bittorrent/session.h

```diff
@@ -445,6 +445,8 @@ namespace BitTorrent
         void setTrackerFilteringEnabled(bool enabled);
         QStringList bannedIPs() const;
         void setBannedIPs(const QStringList &newList);
+        int AIOThreads() const;
```

Comment:
Method names should start with lower case. And I would also use a little more descriptive name, e.g. `asyncIOThreadsCount()`, `setAsyncIOThreadsCount()`, m_asyncIOThreadsCount etc.

Reply:
Thanks for advice!!!